### ISSUE TITLE
“Close” graphical button lacks an accessible name and a role of button [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#191]
The “close” graphical buttons found in the top left corner of the How it works screens lack accessible names and do not convey a role of button. Nothing is announced when the buttons are focused using VoiceOver.

![Close graphical button focused with VoiceOver and nothing is announced](https://user-images.githubusercontent.com/4437084/91864327-14d74080-ec3e-11ea-8391-b34c23e0f625.png)

Use the accessibilityLabel attribute on either the image or the interactive container element to provide an accessible name. Use the accessibilityRole attribute on the interactive container element (e.g., TouchableOpacity, Pressable, etc.) to convey the button role to VoiceOver users.

```javascript
<TouchableOpacity onPress={…} accessibilityLabel='Close How it Works' accessibilityRole='button'>
  <Image source={…} />
</TouchableOpacity>
```

**Comment by willslone** [comment]
I retested and can confirm the issue has been resolved.
